app/views/families/index.html.erb

```diff
@@ -12,8 +12,7 @@
     <% if notice %>
       <div class="alert alert-dismissible alert-success">
         <button class="close" type="button" data-dismiss="alert">×</button>
-        <p><%= notice %><a class="alert-link" href="#"></p>
-        </a></p>
+        <p><%= notice %><a class="alert-link" href="#"></a></p>
```

Comment:
I assume this indentation fix was something your editor did automatically on save? It makes me wonder if we should add an [`.editorconfig`](https://editorconfig.org/) file to the repo (It also makes me wonder if editorconfig would catch this sort of thing)

Nonetheless, thanks for the cleanup!

Reply:
removed the extra closing `p` tag there.

Reply:
Ahh I see it now. Was hard to see what was going on.